cypress/pages/text_box/locators.mjs

```diff
@@ -0,0 +1,9 @@
+export const USER_NAME_LOCATOR = '#userName';
+export const USER_EMAIL_LOCATOR = '#userEmail';
+export const CURRENT_ADDRESS_LOCATOR = '#currentAddress';
+export const PERMANENT_ADDRESS_LOCATOR = '#permanentAddress';
+export const SUBMIT_BUTTON_LOCATOR = '#submit';
+export const OUTPUT_USER_NAME_LOCATOR = '#output #name';
+export const OUTPUT_USER_EMAIL_LOCATOR = '#output #email';
+export const OUTPUT_CURRENT_ADDRESS_LOCATOR = '#output #currentAddress';
+export const OUTPUT_PERMANENT_ADDRESS_LOCATOR = '#output #permanentAddress';
```

Comment:
U have a duplicated selectors here and cypress/pages/student_registration_form/locators.mjs. U are able to create some common selectors folder for them

Reply:
Done